### PR TITLE
[CORL-478] Porting over Danish translations to v5

### PR DESCRIPTION
**Pull request by kgardnr**
## What does this PR do?
- Adds Danish for v5

## How do I test this PR?

- Add LOCALE env variable "da"
- See copy reflected in Danish
- Corrections to typos and grammar welcome!

### CODE REVIEW

File: src/locales/da/install.ftl [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+install-permittedDomains-title = Permitted Domains
+install-permittedDomains-description = Enter the domains you would like to permit for { -product-name }, e.g. your local, staging and production environments (ex. localhost:3000, staging.domain.com, domain.com).
+install-permittedDomains-permittedDomains = Permitted Domains
+install-permittedDomains-permittedDomainsTextField =
+  .placeholder = Domains
+install-permittedDomains-permittedDomainsDescription = Insert domains separated by comma
+
+install-finalStep-description = Thanks for installing { -product-name }! We sent an email to verify your email address. While you finish setting up the account, you can start engaging with your readers now.
+install-finalStep-goToTheDocs = Go to the Docs
+install-finalStep-goToAdmin = Go to Admin
```
Comment (wyattjoh) [inline src/locales/da/install.ftl]: Remove english from file

File: src/locales/da/auth.ftl [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+addEmailAddress-addEmailAddressButton =
+  Add Email Address
+
+## Create Password
+createPassword-createPasswordHeader = Create Password
+createPassword-whatItIs =
+  To protect against unauthorized changes to your account,
+  we require users to create a password.
+createPassword-createPasswordButton =
+  Create Password
```
Comment (wyattjoh) [inline src/locales/da/auth.ftl]: Remove english from file

**Review by wyattjoh — CHANGES_REQUESTED** [review]

File: src/locales/da/install.ftl (hunk 1 above)
Comment (kgardnr) [reply src/locales/da/install.ftl]: Removed for now, as not essential

File: src/locales/da/auth.ftl (hunk 2 above)
Comment (kgardnr) [reply src/locales/da/auth.ftl]: Removed for now, as not essential

File: src/locales/da/admin.ftl [hunk 3]
```diff
@@ -0,0 +1,669 @@
...
+      *[other] år
+    }
+    *[other] unknown unit
+  }
+
+## Create Username
+
+createUsername-createUsernameHeader = Lav brugernavn
+createUsername-whatItIs =
+  Your username is an identifier that will appear on all of your comments.
```
Comment (wyattjoh) [inline src/locales/da/admin.ftl]: English!

File: src/locales/da/admin.ftl [hunk 4]
```diff
@@ -0,0 +1,669 @@
...
+  .label = Publikum
+community-filter-optGroupOrganization =
+  .label = Organisation
+community-filter-search = Søg
+community-filter-showMe = Vis mig
+community-filter-allRoles = Alle roller
+community-filter-allStatuses = Alle statuer
+ 
+community-column-username = Brugernavn
+community-column-email = Email
```
Comment (wyattjoh) [inline src/locales/da/admin.ftl:550]: English!

File: src/locales/da/admin.ftl [hunk 5]
```diff
@@ -0,0 +1,669 @@
...
+community-column-memberSince = Medlem siden
+community-column-role = Rolle
+community-column-status = Status
```
Comment (wyattjoh) [inline src/locales/da/admin.ftl:553]: English!

File: src/locales/es/stream.ftl [hunk 6]
```diff
@@ -237,3 +237,4 @@ configure-openStream-description =
 configure-openStream-openStream = Abrir los comentarios
 
 comments-tombstone-ignore = Este comentario está oculto porque ignoraste a {$username}
+>>>>>>> c979dfd7d52721a9f1a3a5365c61192a3edd3eab
```
Comment (wyattjoh) [inline src/locales/es/stream.ftl]: Looks like a merge conflict.

**Review by wyattjoh — CHANGES_REQUESTED** [review]

File: src/locales/da/admin.ftl (hunk 4 above)
Comment (kgardnr) [reply src/locales/da/admin.ftl:550]: They have some common words - Email, Status, a few others, so these are good

File: src/locales/da/admin.ftl (hunk 5 above)
Comment (kgardnr) [reply src/locales/da/admin.ftl:553]: Same as above!

**Review by wyattjoh — APPROVED** [review]

**Pull request merged**